rest/model/image.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/thirdparty"
+	"github.com/evergreen-ci/utility"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIPackage is the model to be returned by the API whenever packages are fetched
+
+type APIPackage struct {
+	Name    *string `json:"name"`
+	Manager *string `json:"manager"`
+	Version *string `json:"version"`
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIImage is the model to be returned by the API whenever images are fetched
+
+type APIImage struct {
+	ID           *string      `json:"id"`
+	AMI          *string      `json:"ami"`
+	Distros      []APIDistro  `json:"distros"`
+	Kernel       *string      `json:"kernel"`
+	LastDeployed time.Time    `json:"last_deployed"`
+	Name         *string      `json:"name"`
+	Packages     []APIPackage `json:"packages"`
```

Comment:
You can remove these two since they'll be determined in the resolvers
```suggestion
	Kernel       *string      `json:"kernel"`
	LastDeployed time.Time    `json:"last_deployed"`
	Name         *string      `json:"name"`
```